static/js/network.js

```diff
@@ -80,6 +90,27 @@ var nodeIds, shadowState, nodesArray, nodes, edgesArray, edges, network;
         addNode(data.id, data.group, data.label, 200, -200);
     }
 
+    function activateCurrentNode(){
+        var selectedNodeIds = network.getSelectedNodes();
+        if(selectedNodeIds==0){return;}
+        setNodeColor(selectedNodeIds[0], activatedColor);
+        $.post('/', {command: 'Activate Node', node: selectedNodeIds[0]});
+    }
+
+    function deactivateNode(nodeId){
+        nodes.update({
+            id: nodeId,
+            color: null
+        });
+    }
+
+    function setNodeColor(nodeId, color){
+        nodes.update({
+            id: nodeId,
+            color: color
```

Comment:
Could you please add a few comments explaining that sort of thing?
It's good that you know where these things are coming from, but anyone who picks this code up later on needs to know too.